new_release_notes.sh

```diff
@@ -90,7 +93,7 @@ then
   echo;echo "Here is how you can generate a file containing the list of github issues :"
   echo      "fire following commands in the gluster code repository cloned via gh cli :"
   echo;echo "     $ script"
-  echo      "     $ gh issue list -L 600 "
+  echo      "     $ gh issue list -s \"all\" -L 1127"
```

Comment:
any reason behind listing 1127 entries?

Reply:
Will try reuse that in another pr. 

I had seen some of the entries were missing if list less number of issues.